modules/custom/ymca_sync/src/Syncer.php

```diff
@@ -19,9 +19,9 @@ class Syncer implements SyncerInterface {
   /**
    * {@inheritdoc}
    */
-  public function proceed() {
-    foreach ($this->steps as $id => $step) {
-      $step['plugin']->{$step['method']}($step['args']);
+  public function proceed(array $options = []) {
+    foreach ($this->steps as $step) {
+      $step['plugin']->{$step['method']}($options);
```

Comment:
Where did the $step['args'] go? Is it not used anymore? Was it used in any syncers?...